src/TesApi.Web/TesEventMessage.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using TesApi.Web.Storage;
+
+namespace TesApi.Web
+{
+    /// <summary>
+    /// Represents the events sent by the node task runner.
+    /// </summary>
+    /// <remarks>This should be transient in DI.</remarks>
+    public class TesEventMessage
```

Comment:
The name does not reflect the purpose of this class. It acts as a message processor, not a data object or entity. A better name could be: RunnerEventsProcessor, RunnerEventsHandler or RunnerEventsConsumer.